base_test.hpp

```diff
@@ -43,7 +43,11 @@
 #ifndef ROC_BANDWIDTH_TEST_BASE_H_
 #define ROC_BANDWIDTH_TEST_BASE_H_
 
+#if(defined(RBT_HSA_VERSION_FLAT) && ((RBT_HSA_VERSION_FLAT) < RBT_HSA_VERSION_FILEREORG))
```

Comment:
Add comments as to what these branches refer to (new scheme and old scheme) and also which one is deprecated and some future ROCm release when it will be removed

Reply:
Done.  Right now hsa will  have header files in /opt/rocm-ver/hsa/include/hsa. and /opt/rcom-ver/include/hsa.
Planning to remove hsa/include/hsa from rocm 6.0 onwards. But in the comment just mentioned future release , similar to warning message from wrapper header files